logging/taillogs/taillogs_test.go

```diff
@@ -72,21 +73,12 @@ func TestTailLogs(t *testing.T) {
 		adminClient.DeleteLog(ctx, logID)
 	}()
 
-	// following implements custom 2 min timeout for running tailLogs() sample
-	success := make(chan int, 1)
-	go func() {
+	// 5 minute timeout and 3 retries
+	testutil.Retry(t, 3, 5*time.Minute, func(r *testutil.R) {
 		// ingest a couple of logs to finish the test
 		err := tailLogs(projectID)
 		if err != nil {
 			t.Errorf("testLogs sample returned error: %v", err)
```

Comment:
```suggestion
			r.Errorf("testLogs sample returned error: %v", err)
```
we want to use `r` here, as calling `t.Errorf` will shortcut the retries.

Reply:
Done.